checkasm/checkasm.h

```diff
@@ -30,7 +30,7 @@
 
 #include "config.h"
 
-#include "ass_bitmap.h"
+#include "ass_bitmap_engine.h"
```

Comment:
left over in the latest push

Reply:
i left it intentionally since regardless of splitting or not, checkasm doesn't use anything from `ass_bitmpa.h`; though if desired i can drop it anyway